python/hail/methods/family_methods.py

```diff
@@ -55,3 +56,172 @@ def trio_matrix(dataset, pedigree, complete_trios=False):
+        - **error** (*String*) -- Readable representation of Mendel error.
+
+    **Second table:** errors per nuclear family. This table contains one row
+    per nuclear family in the dataset. This table closely reflects the structure
+    of the ".fmendel" PLINK format detailed below.
+
+    Columns:
+
+        - **fid** (*String*) -- Family ID.
+        - **father** (*String*) -- Paternal ID.
+        - **mother** (*String*) -- Maternal ID.
+        - **nChildren** (*Int32*) -- Number of children in this nuclear family.
+        - **nErrors** (*Int32*) -- Number of Mendel errors in this nuclear family.
+        - **nSNP** (*Int32*) -- Number of Mendel errors at SNPs in this nuclear
+          family.
+
+    **Third table:** errors per individual. This table contains one row per
+    individual in the dataset, including founders. This table closely reflects
+    the structure of the ".imendel" PLINK format detailed below.
+
+    Columns:
+
+        - **s** (*String*) -- Sample ID (key column).
+        - **fid** (*String*) -- Family ID.
+        - **nErrors** (*Int64*) -- Number of Mendel errors found involving this
+          individual.
+        - **nSNP** (*Int64*) -- Number of Mendel errors found involving this
+          individual at SNPs.
+        - **error** (*String*) -- Readable representation of Mendel error.
+
+    **Fourth table:** errors per variant. This table contains one row per
+    variant in the dataset.
+
+    Columns:
+
+        - **v** (*Variant*) -- Variant (key column).
+        - **nErrors** (*Int32*) -- Number of Mendel errors in this variant.
+
+    **PLINK Mendel error formats:**
+
+        - ``*.mendel`` -- all mendel errors: FID KID CHR SNP CODE ERROR
+        - ``*.fmendel`` -- error count per nuclear family: FID PAT MAT CHLD N
+        - ``*.imendel`` -- error count per individual: FID IID N
+        - ``*.lmendel`` -- error count per variant: CHR SNP N
+
+    In the PLINK formats, **FID**, **KID**, **PAT**, **MAT**, and **IID** refer
+    to family, kid, dad, mom, and individual ID, respectively, with missing
+    values set to ``0``. SNP denotes the variant identifier ``chr:pos:ref:alt``.
+    N is the error count. CHLD is the number of children in a nuclear family.
+
+    The CODE of each Mendel error is determined by the table below, extending
+    the `Plink classification
+    <https://www.cog-genomics.org/plink2/basic_stats#mendel>`__.
+
+    Those individuals implicated by each code are in bold.
+
+    The copy state of a locus with respect to a trio is defined as follows,
+    where PAR is the `pseudoautosomal region
+    <https://en.wikipedia.org/wiki/Pseudoautosomal_region>`__ (PAR) defined by
+    the reference genome of the locus.
+
+    - HemiX -- in non-PAR of X, male child
+    - HemiY -- in non-PAR of Y, male child
+    - Auto -- otherwise (in autosome or PAR, or female child)
+
+    Any refers to :math:`\{ HomRef, Het, HomVar, NoCall \}` and ! denotes
```

Comment:
Use `~` for complement, that works with expr!

Reply:
Also, can you put some kind of formatting around `Any`? Took me a couple of tries to parse this sentence an I think that would help.